azext_edge/edge/providers/edge_api/clusterconfig.py

```diff
@@ -10,3 +10,9 @@
 CLUSTER_CONFIG_API_V1 = EdgeResourceApi(
     group="clusterconfig.azure.com", version="v1", moniker="clusterconfig", label="microsoft-iotoperations"
 )
+
+CLUSTER_CONFIG_API_V1B1 = EdgeResourceApi(
+    group="clusterconfig.azure.com", version="v1beta1", moniker="clusterconfig", label="microsoft-iotoperations"
```

Comment:
is this needed anymore?

Reply:
Correct, it was introduced in this PR solely for adding the new billing API content to support bundles, which we then removed.

I was conflicted leaving it here because we may eventually add this API's components in the future, but for now I think it's fine to remove